e2e/scientists_queries.yaml

```diff
@@ -846,18 +846,18 @@ queries:
       - selected: ["?a", "?b", "?c"]
 
 
-  - query: nonexisting-values
+  - query: new-values
     type: no-text
     sparql: |
-      SELECT ?a WHERE {
-        VALUES ?a {"obscure Literal"@xf <Albert_Einstein> <Non_Exisiting_Scientist>}
+      SELECT ?x ?y ?z WHERE {
+        VALUES (?x ?y ?z) { (<Albert_Einstein> <Ulm> 1879) (<Albert_Zweistein> <Zwulm> 2001) }
+        VALUES ?x { <Albert_Einstein> }
       }
     checks:
-      - num_cols: 1
+      - num_cols: 3
       - num_rows: 1
-      - selected: ["?a"]
-      - contains_row: ["<Albert_Einstein>"]
-      - contains_warning: ["The word \"obscure Literal\"@xf", "The word <Non_Exisiting", "Ignored 2 rows"]
+      - selected: ["?x", "?y", "?z"]
+      - contains_row: ["<Albert_Einstein>", "<Ulm>", 1879]
```

Comment:
I can't have new words in both VALUES clauses because then we would have a JOIN with two non-empty vocabularies, which we currently don't allow. I have now simply switched `<Ulm>` and `<Zwulm>` such that `<Zwulm>` becomes part of the output (assuming that that is what you mean by "export").